subcommand/sync-catalog/command.go

```diff
@@ -47,8 +50,13 @@ func (c *Command) init() {
 	c.flags.StringVar(&c.flagConsulServicePrefix, "consul-service-prefix",
 		"", "A prefix to prepend to all services written to Consul from AWS. "+
 			"If this is not set then services will have no prefix.")
-	c.flags.StringVar(&c.flagAWSPullInterval, "aws-pull-interval",
-		"30s", "The interval between fetching from AWS CloudMap. "+
+	c.flags.StringVar(&c.flagAWSDeprecatedPullInterval, "aws-pull-interval",
+		DefaultPollInterval, "[DEPRECATED] The interval between fetching from AWS CloudMap. "+
```

Comment:
Worth saying something like "[DEPRECATED] - use -aws-poll-interval which has the same behaviour"

Reply:
Ah it's below. Probably fine then.